site/docs/index.md

```diff
@@ -6,50 +6,68 @@ sidebar_position: 1
-- **AWS Request Signing**: Supports external processing for secure communication with AWS-hosted LLM/AI services.
-
-Documentation for installation, setup, and contribution guidelines is included to help new users and contributors get started easily.
+  This also controls LLM/AI backend usage using token-per-second (TPS) policies to prevent overuse.
+- **Multi-Upstream Provider Support for LLM/AI Services**: The ability to receive requests in the
+  format of one LLM provider and route them to different upstream providers, ensuring compatibility
+  with their expected formats. This is made possible through built-in transformation capabilities that
+  adapt requests and responses accordingly.
+- **AWS Request Signing**: Supports external processing for secure communication with AWS-hosted
+  LLM/AI services.
 
+Documentation for installation, setup, and contribution guidelines is included to help new users and
+contributors get started easily.
 
 ## **Community Collaboration**
 
-Weekly community meetings are held every Thursday to discuss updates, address issues, and review contributions.
+[Weekly community meetings][meeting-notes] are held every Thursday to discuss updates, address
+issues, and review contributions.
 
 ## **Architecture Overview**
 
-
 ## **Get Involved**
 
 We welcome community contributions! Here's how you can participate:
-- Attend the weekly community meetings to stay updated and share ideas.
+
+- Attend the [weekly community meetings][meeting-notes] to stay updated and share ideas.
 - Submit feature requests and pull requests via the GitHub repository.
-- Join discussions in the #envoy-ai-gateway Slack channel.
+- Join discussions in the [#envoy-ai-gateway] Slack channel.
 
-Refer to the contribution guide in the GitHub repository for detailed instructions on setting up your environment and contributing.
+Refer to [this contributing guide][contributing.md] for detailed instructions on setting up your
+environment and contributing.
 
 ---
 
-The **Envoy AI Gateway** addresses the growing demand for secure, scalable, and efficient AI/LLM traffic management. Your contributions and feedback are key to its success and to advancing the future of AI service integration.
+The **Envoy AI Gateway** addresses the growing demand for secure, scalable, and efficient AI/LLM
+traffic management. Your contributions and feedback are key to its success and to advancing the
+future of AI service integration.
+
+[meeting-notes]: https://docs.google.com/document/d/10e1sfsF-3G3Du5nBHGmLjXw5GVMqqCvFDqp_O65B0_w
+[#envoy-ai-gateway]: https://envoyproxy.slack.com/archives/C07Q4N24VAA
+[contributing.md]: https://github.com/envoyproxy/ai-gateway/blob/main/CONTRIBUTING.md
```

Comment:
links I added that are new content on this page and not Markdown formatting changes

Reply:
thank you for the follow up comment!